src/components/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
```

Comment:
Kindly do well to take this away, since it is required for this milestone that the initial state should contain only an empty array of categories😉.
```suggestion

```